src/Symfony/Component/Translation/TranslatorFallbackInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation;
+
+/**
+ * @author Bozhidar Hristov <warxcell@gmail.com>
+ */
+interface TranslatorFallbackInterface extends TranslatorInterface
+{
+    /**
+     * @return string[] $locales
+     */
+    public function getFallbackLocales();
+
+    /**
+     * @param string[] $locales
+     *
+     * @throws \InvalidArgumentException If a locale contains invalid characters
+     */
+    public function setFallbackLocales(array $locales);
```

Comment:
I'm not sure this method needs to be in the interface. Configuring the instance does not need to belong in the API.

Reply:
Then why `TranslatorInterface` has ability to `setLocale` ?

Reply:
The locale can change throughout the request. But I agree with @stof that this is probably not needed for the fallback locales.